ui/library/src/utils/scroll.ts

```diff
@@ -4,7 +4,7 @@ import { PageRotation } from '../utils/rotate';
 // Each page div is ID'd according to page index
 // e.g. reader_pg_0, reader_pg_1, etc.
 export const PAGE_NAV_TARGET_ID_ROOT = 'reader_pg_';
-export const SCROLLABLE_TARGET_DIV_CLASSNAME = 'reader__page-list';
+export const SCROLLABLE_TARGET_DIV_CLASSNAME = 'react-pdf__Document';
```

Comment:
We shouldn't need to define a class name for this. We should be able to get a ref of the page we want to scroll to, then find it's closest scrollable parent. That allows someone to set the scrolling in their CSS without having to define something in JS.